build.gradle

```diff
@@ -124,21 +131,29 @@ task wrapper(type: Wrapper) {
     gradleVersion = '3.0'
 }
 
-pluginBundle {
-    website = 'https://github.com/allegro/axion-release-plugin'
-    vcsUrl = 'https://github.com/allegro/axion-release-plugin'
-    description = 'Release and version management plugin.'
-    tags = ['release', 'version']
-
-    plugins {
-        release {
-            id = 'pl.allegro.tech.build.axion-release'
-            displayName = 'axion-release-plugin'
-        }
-    }
-
-    mavenCoordinates {
-        groupId = 'pl.allegro.tech.build'
-        artifactId = 'axion-release-plugin'
-    }
+//pluginBundle {
+//    website = 'https://github.com/allegro/axion-release-plugin'
+//    vcsUrl = 'https://github.com/allegro/axion-release-plugin'
+//    description = 'Release and version management plugin.'
+//    tags = ['release', 'version']
+//
+//    plugins {
+//        release {
+//            id = 'pl.allegro.tech.build.axion-release'
+//            displayName = 'axion-release-plugin'
+//        }
+//    }
+//
+//    mavenCoordinates {
+//        groupId = 'pl.allegro.tech.build'
+//        artifactId = 'axion-release-plugin'
+//    }
+//}
+
+publishing {
+	repositories {
+		maven {
+			url uri('file://' + new File(System.getProperty('user.home'), '.m2/repository').absolutePath)
```

Comment:
Is that part of your PR intended to be committed (together with `maven-publish`?

Btw, if you just want to install locally built version of Axion `./gradlew install` should be enough - without any modifications in `build.gradle`.